src/CarrierLogo/README.md

```diff
@@ -0,0 +1,32 @@
+# CarrierLogo
+To implement CarrierLogo component into your project you'll need to add the import:
+```jsx
+import CarrierLogo from "@kiwicom/orbit-components/lib/CarrierLogo";
+```
+After adding import into your project you can use it simply like:
+```jsx
+<CarrierLogo carriers={Carrier} />
+```
+## :rocket: Props
+Table below contains all type of the props available in CarrierLogo component.
+
+| Name          | Type                             | Default         | Description                      |
+| :------------ | :------------------------------- | :-------------- | :------------------------------- |
+| **carriers**  | [`Carrier[]`](#airplane️-carrier) |                 | The content of the CarrierLogo, passed as array of objects.
+| size          | [`enum`](#enum)                  | `"large"`       | The size of the CarrierLogo.
```

Comment:
Please add **functional specs** for prop `size` - it doesn't apply when more than one carrier is in `carriers`.